src/SFA.DAS.ApprenticeAccounts.UnitTests/Application/Queries/ApprenticeSync/WhenHandlingSyncApprentices.cs

```diff
@@ -0,0 +1,189 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.ApprenticeAccounts.Application.Queries.SyncApprenticeAccountsQuery;
+using SFA.DAS.ApprenticeAccounts.Data;
+using SFA.DAS.ApprenticeAccounts.Data.Models;
+using SFA.DAS.ApprenticeAccounts.DTOs.Apprentice;
+using SFA.DAS.Testing.AutoFixture;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Mail;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApprenticeAccounts.UnitTests.Application.Queries.ApprenticeSync;
+public class WhenHandlingSyncApprentices
+{
+    [Test]
+    [RecursiveMoqAutoData]
+    public async Task WithNullDateThenExpectedArrayOfApprenticeSyncDtoReturned(
```

Comment:
Looking at the existing test name convention, the test name should be `AndNullDate_Then...`